src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -137,6 +138,33 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, correlat
     api_instance = kube_client.CoreV1Api()
     node_api_response = utils.validate_node_api_response(api_instance, None)
 
+    # Pre onboarding checks
+    try:
+        absolute_path = os.path.abspath(os.path.dirname(__file__))
+        kubectl_client_location = install_kubectl_client()
+        helm_client_location = install_helm_client()
+        release_namespace = get_release_namespace(kube_config, kube_context, helm_client_location)
```

Comment:
why do we need release_namespace? release doesn't even exist right?

Reply:
did you check what this is returning?

Reply:
it returns "default"
anyways , removing the release namespace since its not needed